### PR TITLE
Feat/touchpad

### PR DESCRIPTION
**Pull request by PhilippAubert**
Touchpad-functions added: Start on touch, stop on release, change Oscillator pitch on touch move.

### CODE REVIEW

File: src/components/Oscillators.js [hunk 1]
```diff
@@ -85,22 +85,18 @@ export default function Oscillators({
           >
             <Square />
           </button>{" "}
-          <button
-            id="sine"
-            className={"Change"}
-            onClick={handleChangeOsc2Type}
-          >
+          <button id="sine" className={"Change"} onClick={handleChangeOsc2Type}>
```
Comment (mpagels) [inline src/components/Oscillators.js:88]: Do you need the `{` and the `}` here? after `className`
```suggestion
          <button id="sine" className="Change" onClick={handleChangeOsc2Type}>
```

**Review by mpagels — APPROVED** [review]
Incredible work Philipp. A lot of the time I don't understand what is happening. So I decided to look for stuff I understand. ;-). Found some tiny things. Pls review.

Again, awesome work. Keep pushing.

**Review by namirsab — CHANGES_REQUESTED** [review]
Good job @PhilippAubert 
i had a lot of fun playing around with the snyth ;)
Please check the comments I've left. I think they might be helpful

File: src/components/Oscillators.js (hunk 1 above)
Comment (PhilippAubert) [reply src/components/Oscillators.js]: I tried out both - I don't need the curly braces 😃

**Pull request merged**